config_models/tests/test_admin.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Purpose of this test is to just test admin.py file syntax and avoid any warnings or issues.